packages/proto-loader/src/index.ts

```diff
@@ -395,6 +395,15 @@ export function loadSync(
   return createPackageDefinition(root, options!);
 }
 
+export function fromJSON(
+  json: Protobuf.INamespace
+): PackageDefinition {
+  const newRoot: Protobuf.Root = new Protobuf.Root();
+  const loadedRoot = Protobuf.Root.fromJSON(json, newRoot);
+  loadedRoot.resolveAll();
+  return createPackageDefinition(newRoot, {});
```

Comment:
The options should be an argument, instead of forcing the default values.

Reply:
Thank you for your reply. This `{}` indeed a bit ugly, but `options` arg is already included in the definition of json description, I think it maybe unnecessary to add additional argument.